spec/spec.md

```diff
@@ -406,21 +406,27 @@ specified by a [[ref:Feature]]:
     earliest termination of evaluation. If the `fields` property is present,
     its value ****MUST**** be an array of objects composed as follows, unless
     otherwise specified by a feature:
-        - The _fields object_ ****MUST**** contain a `path` property. The value
-          of this property ****MUST**** be an array of one or more
-          [JSONPath](https://goessner.net/articles/JsonPath/) string
-          expressions (as defined in the
-          [JSONPath Syntax Definition](#jsonpath-syntax-definition) section)
-          that select a target value from the input. The array ****MUST****
-          be evaluated from 0-index forward, breaking as soon as a _Field
-          Query Result_ is found (as described in
-          [Input Evaluation](#input-evaluation)), which will be used for the
-          rest of the entry's evaluation. The ability to declare multiple
+        - The _fields object_ ****MUST**** contain either a `pointer` or `path`
+          property, array-valued. The array ****MUST**** be evaluated from 
+          0-index forward, breaking as soon as a _Field Query Result_ is found
+          (as described in [Input Evaluation](#input-evaluation)), which will
+          be used for the rest of the entry's evaluation. The ability to
+          declare multiple
           expressions in this way allows the [[ref:Verifier]] to account for
           format differences - for example: normalizing the differences in
           structure between JSON-LD/JWT-based
           [Verifiable Credentials](https://www.w3.org/TR/vc-data-model/) and
           vanilla JSON Web Tokens (JWTs) [[spec:rfc7519]].
+              - If provided, the `pointer` property ****MUST**** be an array of
```

Comment:
I think I prefer `path` over `pointer` as a property-name, but we need to consider if that has negative implication towards forward (or backward) compatibility.